docs/migration/0.6_c_qubit_converter.rst

```diff
@@ -0,0 +1,268 @@
+
+   converter = QubitConverter(ParityMapper(), two_qubit_reduction=True)
+
+   reduced_op = converter.convert(hamiltonian, num_particles=problem.num_particles)
+
+   for pauli, coeff in sorted(reduced_op.primitive.label_iter()):
+       print(f"{coeff.real:+.8f} * {pauli}")
+
+.. testoutput::
+
+    -1.05237325 * II
+    +0.39793742 * IZ
+    +0.18093120 * XX
+    -0.39793742 * ZI
+    -0.01128010 * ZZ
+
+The equivalent code now looks like the following:
+
+.. testcode::
+
+   mapper = ParityMapper(num_particles=problem.num_particles)
+
+   reduced_op = mapper.map(hamiltonian)
+
+   for pauli, coeff in sorted(reduced_op.primitive.label_iter()):
+       print(f"{coeff.real:+.8f} * {pauli}")
+
+.. testoutput::
+
+    -1.05237325 * II
+    +0.39793742 * IZ
+    +0.18093120 * XX
+    -0.39793742 * ZI
+    -0.01128010 * ZZ
+
+Z2Symmetries
+------------
+
+Finally, the ``QubitConverter`` class also supported further reduction of qubit resources by
+exploiting :class:`~qiskit.opflow.Z2Symmetries`. Although we did extend the deprecated class to also
+support the updated implementation in :class:`~qiskit.quantum_info.analysis.Z2Symmetries`, you
+should now use the new :class:`~qiskit_nature.second_q.mappers.TaperedQubitMapper` instead.
+
+In the past, you would have enabled this like so:
+
+.. testcode::
+
+   mapper = JordanWignerMapper()
+   converter = QubitConverter(mapper, z2symmetry_reduction="auto")
+
+which would then later use
+:meth:`~qiskit_nature.second_q.problems.BaseProblem.symmetry_sector_locator` to find the symmetry
+sector of the Hilbert space in which the solution of your problem lies. This was only supported by
+the :class:`~qiskit_nature.second_q.problems.ElectronicStructureProblem`. Below is a quick example:
+
+.. testcode::
+
+   tapered_op = converter.convert(
+       hamiltonian,
+       num_particles=problem.num_particles,
+       sector_locator=problem.symmetry_sector_locator,
+   )
+
+   for pauli, coeff in sorted(tapered_op.primitive.label_iter()):
+       print(f"{coeff.real:+.8f} * {pauli}")
+
+.. testoutput::
+
+    -1.04109314 * I
+    +0.18093120 * X
+    -0.79587485 * Z
+
+Now, all you need to do is the use the
+:meth:`~qiskit_nature.second_q.problems.BaseProblem.get_tapered_mapper` method and provide the
+original mapper which you would like to wrap:
+
+.. testcode::
+
+   tapered_mapper = problem.get_tapered_mapper(mapper)
+
+   tapered_op = tapered_mapper.map(hamiltonian)
+
+   for pauli, coeff in sorted(tapered_op.primitive.label_iter()):
+       print(f"{coeff.real:+.8f} * {pauli}")
+
+.. testoutput::
+
+    -1.04109314 * I
+    +0.18093120 * X
+    -0.79587485 * Z
+
+If you were not using the automatic ``Z2Symmetries`` detection but instead provided a custom
+symmetry sector, you can construct your :class:`~qiskit_nature.second_q.mappers.TaperedQubitMapper`
+instance directly. Be sure to check out its documentation for more details.
+
+PauliSumOp vs. SparsePauliOp
```

Comment:
I commented on this earlier. The setting is really only around output from the classes, like the mapper where this applies. The input cases now take either right, like the solve() in QEOM but where usage of PauliSumpOp as input is deprecated and will be removed. In general, unless you relied on the type being output by say a mapper, already using SparsePauliOp would be recommended.